samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java

```diff
@@ -2863,6 +2863,8 @@ private String getTimerId(DeliveryBean delivery){
   
   private long getSeed(SectionDataIfc sectionData, DeliveryBean delivery, long userSeed) {
 	  long seed = userSeed;
+	  String sectionRandomizationSeed = sectionData.getSectionMetaDataByLabel(SectionDataIfc.RANDOMIZATION_SEED);
+	  if (StringUtils.isNotBlank(sectionRandomizationSeed)) seed += Long.parseLong(sectionRandomizationSeed);
```

Comment:
Please, include a comment.

// If the section has a previous seed we must use it to use the same order.

Also use brackets here:

if (StringUtils.isNotBlank(sectionRandomizationSeed)) {
   seed += Long.parseLong(sectionRandomizationSeed);
}